rust_src/src/data.rs

```diff
@@ -49,3 +52,17 @@ pub fn indirect_function_lisp(object: LispObject, _noerror: LispObject) -> LispO
     }
     return result;
 }
+
+/// Return SYMBOL's value.  Error if that is void.  Note that if
+/// `lexical-binding' is in effect, this returns the global value
+/// outside of any lexical scope.
+#[lisp_fn]
+pub fn symbol_value(symbol: LispObject) -> LispObject {
```

Comment:
`symbols.rs` would be a better place for this function.

Reply:
OK, didn't realize breaking symmetry with the C code was an option. Will do!